app/javascript/components/Admin/Products/Purchases/index.tsx

```diff
@@ -8,15 +8,15 @@ import { type ProductPurchase } from "./Purchase";
 
 type AdminProductPurchasesProps = {
   product_id: number;
-  is_affiliate_user?: boolean;
+  isAffiliateUser?: boolean;
   user_id: number | null;
```

Comment:
Would be nice to make the other props camelCase while we're here (didn't notice those, my bad)

Reply:
I've made that change: https://github.com/antiwork/gumroad/pull/1862/commits/2941fb353c24ba17b01b6fb4fe56f380020624a3